api/cas-server-core-api-configuration-model/src/test/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,134 @@
+{
+  "properties": [
```

Comment:
This does not belong here, does it?

Reply:
Not entirely sure why but the test fails without the file in the same jar as the test.

Reply:
File removed and it's passing locally...